### PR TITLE
Removed per user podcasts

### PR DESCRIPTION
**Pull request by brian-doherty**

### CODE REVIEW

**Review by sentriz — COMMENTED** [review]
looks good

@lxea  any thoughts on this?

File: README.md [hunk 1]
```diff
@@ -17,7 +17,6 @@
 - browsing by tags (using [taglib](https://taglib.org/) - supports mp3, opus, flac, ape, m4a, wav, etc.)  
 - on-the-fly audio transcoding and caching (requires [ffmpeg](https://ffmpeg.org/)) (thank you [spijet](https://github.com/spijet/))  
 - jukebox mode (thank you [lxea](https://github.com/lxea/))  
-- support for per-user podcasts (thank you [lxea](https://github.com/lxea/))
```
[the remark this answers is not in the document]
Comment (brian-doherty) [reply README.md:20]: OK by me.

**Comment by sentriz** [comment]
squashed and merged :+1: 
also I removed the logic to not delete a podcast from the filesystem if there are still users using it. since we don't need that now

image

also @brian-doherty in case you didn't know, you can base your branches on master to avoid the messy history :grin: 

```
git fetch origin/master
git checkout -b new-feature origin/master
```  

anyway thank you kindly!

**Comment by brian-doherty** [comment]
Great question. There are two reasons that I proposed this removal:

1) The Subsonic API is quite clear that podcasts are per-server -- there are several references to that fact in the API doc (http://www.subsonic.org/pages/api.jsp) and podcast addition is an admin privilege.
2) They were never properly implemented in the first place -- in particular, if two users tried to add the same podcast everything would collide on the FS, which was problematic, especially if one of them decided to delete something.

**Comment by benyanke** [comment]
That's pretty reasonable then. I would love per-user podcasts, but that's evidently not part of the subsonic API spec, so I can't knock someone for being faithful to the API spec, since this is just a reimplementation of it. Thanks for the info!

**Pull request merged**